src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1338,13 +1440,13 @@ private void doEncodeSaslOutcome(
             AmqpSaslInitFW saslInit)
         {
             final AmqpSaslOutcomeFW saslOutcome =
-                amqpSaslOutcomeRW.wrap(frameBuffer, FRAME_HEADER_SIZE, frameBuffer.capacity())
+                amqpSaslOutcomeRW.wrap(frameBuffer, FRAME_HEADER_SIZE + PERFORMATIVE_SIZE, frameBuffer.capacity())
                     .code(OK)
                     .build();
 
             final AmqpSaslFrameHeaderFW saslFrameHeader =
                 amqpSaslFrameHeaderRW.wrap(frameBuffer, 0, frameBuffer.capacity())
-                    .size(FRAME_HEADER_SIZE + saslOutcome.sizeof())
+                    .size(FRAME_HEADER_SIZE + PERFORMATIVE_SIZE + saslOutcome.sizeof())
```

Comment:
`sasl` frames don't have a `performative`, that just for `plain` frames, right?

Reply:
well `sasl` frames don't have a performative but it still has 3 byte descriptor

Reply:
Then we need a different constant to make the code more readable please.